src/components/attendee/VideoHuddle/VideoHuddle.module.scss

```diff
@@ -57,6 +57,13 @@ $-control-size: 32px;
   overflow-x: scroll;
   overflow-y: hidden;
   display: flex;
-  justify-content: center;
   gap: space.gap(1);
+
+  // use pseudoelements instead of justify-content:center to avoid hiding of the first child elements on content overflow
+  // @see https://stackoverflow.com/a/46059525
```

Comment:
Were other solutions considered? 
If so, it might be good to add the conclusion of those in the comment above `@see`

This is a solution from 2017 so the question is why should it be preferred over `display: flex` or `display: grid` with the proper `align-`, `justify-` and `place-` properties?